deploy/template.yaml

```diff
@@ -0,0 +1,1339 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >-
+  This creates the necessary components to deploy the Account Management
+  Frontend onto ECS Fargate.
+  Account Management Frontend can be invoked via the CloudFront Distribution, 
+  use the CloudFrontDistributionEndpoint URL exported by this stack.
+  The ingress route in summary is: 
+  CloudFront -> WAF -> API Gateway -> VPC link -> Private ALB -> ECS Fargate
+Transform:
+  - AWS::Serverless-2016-10-31
+
+# Must deploy the container on it's own first, having a successful cluster/container spun up
```

Comment:
We know about this, can come along and uncomment after a first successful deploy to all envs.

Reply:
created a Jira - https://govukverify.atlassian.net/browse/ATB-119